Qadir__Work/HTML/Assignment6/style.css

```diff
@@ -0,0 +1,75 @@
+h1 {
+    font-size: x-large;
+}
+
```

Comment:
remove extra line

Reply:
I think i don't want to remove extra lines, Because Professional Programmer doing like that so i am doing same way.

Reply:
ok